<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LaunchDialog</class>
 <widget class="QDialog" name="LaunchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="InfoText">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>451</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>Launching instance...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="LaunchJupyterLabButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Launch Jupyter Lab</string>
   </property>
  </widget>
  <widget class="QPushButton" name="TerminateInstanceButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>200</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Terminate Instance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RefreshConnectionButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh Connection</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
